Lib/urllib/parse.py

```diff
@@ -253,62 +253,58 @@ def _hostinfo(self):
 _ParseResultBase = namedtuple(
     'ParseResult', 'scheme netloc path params query fragment')
 
-_DefragResultBase.__doc__ = """
+_DefragResultBase.__doc__ = """\
 DefragResult(url, fragment)
 
 A 2-tuple that contains the url without fragment identifier and the fragment
-identifier as a separate argument.
-"""
+identifier as a separate argument."""
 
 _DefragResultBase.url.__doc__ = """The URL with no fragment identifier."""
 
-_DefragResultBase.fragment.__doc__ = """
-Fragment identifier separated from URL, that allows indirect identification of a
-secondary resource by reference to a primary resource and additional identifying
-information.
-"""
+_DefragResultBase.fragment.__doc__ = (
+    "Fragment identifier separated from URL, that allows indirect "
+    "identification of a secondary resource by reference to a primary "
+    "resource and additional identifying information."
+)
```

Comment:
There is not a bug. Docstring can be multiline.

Reply:
As per your comment, I reverted the unnecessary code.
Thank you for your review :)